script/Arcueid/Day 8/08_00_ARC08_2_1.txt

```diff
@@ -145,19 +145,19 @@ The high-pitched voice brings me to a halt.
 [sha:c1520a7ef9282c6ccbe9b848cf5a6d83bbb8b63a]{
 -- Page 126, Offset 19982.
 -- 安藤刑事のものなら無視するつもりだったが、その少年の声は妙に逆らいがたい威厳に満ちていた。
-If it had been Detective Ando's then I would have simply ignored it,
- but that boy's voice was filled with a strange sense of dignity that made it difficult to oppose.
+If it had been Detective Andou's then I would have simply ignored it,
+ but that boy's voice commanded such authority that it strangely difficult to oppose.
 }
 [sha:9f334bc060ef2e224d543d3b209ddcece0a68ba5]{
 -- Page 127, Offset 19983.
 -- 「正義漢も結構だが分を<弁|わきま>えな。人間らしい死に方をしたいんなら、このあたりが引き際だぜ」
-"Being righteous is fine and all, but know your place.
- If you want to die as a human being, then this is your chance to turn back."
+"Playin' hero 'n all's fine, but know your fuckin' place.
+ If you wanna die an ordinary human death then now's the time to quit."
```

Comment:
how authoritative 😳 
> death, then

Reply:
Maybe a bit more clear would be something like "If you want your death to be an ordinary human one, then now's the time to quit"

Reply:
either way would be fine